routetable/route_table.go

```diff
@@ -188,9 +188,34 @@ type RouteTable struct {
 	conntrack         conntrackIface
 	time              timeshim.Interface
 
+	additionalRouteFilters *netlink.Route
+
 	opReporter logutils.OpRecorder
 }
 
+type RouteTableOptions func(rt *RouteTable)
```

Comment:
One object of this type is a single option, so please s/RouteTableOptions/RouteTableOption/